lib/src/cmaf/ham/mapper/dash/mapDashToHam.ts

```diff
@@ -46,13 +52,25 @@ function mapTracks(
 		throw new Error('Error: AdaptationSet is undefined');
 	}
 	const type = getContentType(adaptationSet, representation);
+	const frameRate = getFrameRate(adaptationSet, representation).split(
+		FRAME_RATE_SEPARATOR,
+	);
+	const frameRateNumerator = parseInt(frameRate.at(NUMERATOR) ?? '');
+	const frameRateDenominator = parseInt(frameRate.at(DENOMINATOR) ?? '');
 	if (type === 'video') {
 		return {
 			name: getName(adaptationSet, representation, type),
 			bandwidth: +(representation.$.bandwidth ?? 0),
 			codec: getCodec(adaptationSet, representation),
 			duration: getTrackDuration(segments),
-			frameRate: getFrameRate(adaptationSet, representation),
+			frameRate: {
+				frameRateNumerator: isNaN(frameRateNumerator)
+					? FRAME_RATE_NUMERATOR_30
+					: frameRateNumerator,
+				frameRateDenominator: isNaN(frameRateDenominator)
+					? undefined
```

Comment:
Wouldn't this `undefined` here cause unexpected behaviours? Is there any other possible value for this case?

Reply:
For now it won't cause unexpected behaviors. It's not being used for real calculation in the library, it's just being used for parsing purposes. And we can have just numerator without denominator or both. So I put undefined for checking if there is not denominator then use just nominator. 

But we can change the logic to use 0 instead of undefined and check if denominator is distinct to 0. Let me know what do you prefer, I don't have preferences.